torch/csrc/jit/passes/quantization.cpp

```diff
@@ -939,5 +939,103 @@ graph(%self, %scale, %zero_point, %dtype):
   rewriter.RegisterRewritePattern(pattern, replacement);
   rewriter.runOnGraph(graph, filter);
 }
+
+void FoldPrepackedWeightIntoModule(
+    script::Module& module,
+    const std::string& method_name,
+    const script::Module& wrapper_module) {
+  auto method = module.get_method(method_name);
+  auto graph = method.graph();
+  // Recursively fold prepack in all methods in the call hierarchy
+  std::stack<Block*> blocks_to_visit;
+  blocks_to_visit.push(graph->block());
+  while (!blocks_to_visit.empty()) {
+    Block* b = blocks_to_visit.top();
+    blocks_to_visit.pop();
+    for (Node* n : b->nodes()) {
+      if (n->kind() == prim::CallMethod) {
+        auto module_instance = n->inputs()[0];
+        auto module_method_name = n->s(attr::name);
+        script::Module callee_module = module;
+        if (module_instance->node()->kind() == prim::GetAttr) {
+          auto child_module_name = module_instance->node()->s(attr::name);
+          auto child_module = module.find_module(child_module_name);
```

Comment:
We should go through all submodules, there is no reason to scan IR for call-methods for that.

Reply:
you mean all methods of all submodules? what if there are some methods not used?